src/kirby/components/list/list-item/list-item.component.tns.html

```diff
@@ -0,0 +1,18 @@
+<GridLayout columns="2*, *, 2*">
+    <!-- First column -->
+  <StackLayout class="titles-container" col="0">
+    <Label
+      text="{{title}}"
+      class="list-item-title"
+    ></Label>
+    <Label text="{{subTitle}}" class="list-item-subTitle"></Label>
+  </StackLayout>
+  <!-- middle column -->
+  <StackLayout class="detail-container" col="1" >
+    <Label text="{{detail}}" class="list-item-detail" [ngClass]="{'green': detail > 0, 'red': detail < 0}"></Label>
```

Comment:
Flyt til .ts komponenten, så det kan unit testes:
```suggestion
     <Label text="{{detail}}" class="list-item-detail" [ngClass]="itemDetailCssClass"></Label>
```